tools/class-parse/class-parse.csproj

```diff
@@ -12,11 +12,12 @@
     <ProductVersion>8.0.30703</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <Import Project="..\..\Configuration.props" />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug</OutputPath>
+    <OutputPath>$(UtilityOutputPath)</OutputPath>
```

Comment:
Assuming that you set this once and then the projects get built - if you add a new project which is, say, a level deeper, then the relative path `..\..\bin\$(Configuration)` would be invalid for that project.